test/sample.test.js

```diff
@@ -0,0 +1,18 @@
+var chai = require('chai');

```

Comment:
Please do well to adhere to details of this style guide. [Airbnb Style Guide for Javascript](https://github.com/airbnb/javascript). It addresses the preference of using `const` or `let` for [variables](https://github.com/airbnb/javascript#variables--const).